src/greyVest/TableFooter.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import Pager from './Pager'
+import PagerItem from './PagerItem'
+import PageSize from './PageSize'
+import Flex from './Flex'
+
+let showing = (totalRecords, page, pageSize) => {
+  let min = _.min([totalRecords, pageSize * (page - 1) + 1])
+  let max = _.min([totalRecords, pageSize * page])
+  return min >= max ? max : `${min}-${max}`
+}
+
+let TableFooter = ({
+  page = 1,
+  onChangePage,
+  pageSize,
+  onChangePageSize,
+  pageSizeOptions,
+  totalRecords = 0,
+  hasMore,
+  ...props
+}) => {
+  let pageCount = _.ceil(totalRecords / pageSize)
+  return (
+    <Flex
+      justifyContent="space-between"
+      alignItems="center"
+      style={{ padding: 8 }}
+      {...props}
+    >
+      <PageSize
+        sizeOptions={pageSizeOptions}
+        value={pageSize}
+        onChange={onChangePageSize}
+        style={{ flex: 1 }}
+      />
+      <Flex style={{ flex: 1 }} alignItems="center" justifyContent="center">
+        <Pager value={page} onChange={onChangePage} pageCount={pageCount} />
+        {hasMore && page === pageCount && (
+          <PagerItem
+            style={{ margin: '0 8px', paddingLeft: 12, paddingRight: 12 }}
+            onClick={() => onChangePage(page + 1)}
+          >
+            Load More...
+          </PagerItem>
+        )}
+      </Flex>
+      <span style={{ flex: 1, textAlign: 'right' }}>
+        <b>Showing</b> {showing(totalRecords, page, pageSize)}
+        {!hasMore && ` of ${totalRecords}`}
```

Comment:
It occurs to me that this isn't obvious - it's not intuitive that when skipCount is false, hasMore will always be falsey. We should probably change that and make this check based on `node.skipCount`

Reply:
This is a GreyVest component - a fairly specialized one, but it still shouldn't know more about contexture than it has to.

From the perspective of TableFooter, `hasMore` is just a flag that indicates we shouldn't show ` of {totalRecords}` because `totalRecords` may be incorrect (and TableFooter doesn't care about why it's falsy - it's a flag, so falsy is the default).

Reply:
Understood, but the logic ends up confusing - this would need to be passed a falsely value hasMore even when there is more data when the count is provided. Perhaps another alternative would be to call the prop something like hideCount on the pager which is passed from contexture react based on skipCount